README.md

```diff
@@ -142,25 +142,31 @@ Recomputes embeddings.
 | `save_period`      | int    | Optional     | `20`    | Interval (in number of tracking steps) when tracks are saved to the database.               |
 
 
-## PyInstaller build instructions
-*in progress*
-<!-- Run this to create your virtual environment:
-```
-./setup.sh
-```
+## PyInstaller Build Process
```

Comment:
can we please have 2 targets:
1. make
2. make clean
 

1 should build the project
2 should clean the project